### ISSUE TITLE
Message of ContinuousRuleModifyingEffectImpl is not shown to the user

### ISSUE THREAD

**Issue by LevelX2** [magefree/mage#6112]
The message defined in a ContinuousRuleModifyingEffect is no longer shown to users as before.

E.g. if a card like Rule of Law ("Each player can't casr more than one spell each turn") is on the battlefield, the player trying to cast a spell beyond the first does no longer get the info message defined in the effect. Nothing happens. 

Seems like some modified/wrong handling of the CheckPlayableMode flag caused this.

**Comment by jeffwadsworth** [comment]
Just a side note.  While testing the player.cast() code, I noticed that Illusory Angel's "Cast this spell only if you cast another spell this turn" message popped up correctly.  So apparently they are not all affected.

**Comment by jeffwadsworth** [comment]
Ran across an instance where it does pop up, but it did so incorrectly because I had cast a Lightning Bolt before it.  Anyway, I was using Mindleech Mass and attacked doing damage to the opponent.  The opponent's hand had an Illusory Angel.  I got the message in the attachment below when trying to cast it from the opponent's hand via the Mindleech Mass effect.  I then put an Illusory Angel in my own hand and while it correctly prevented me from casting it if I had not cast another spell before it, the message did not pop up.
![PoppedUpRuleChanging](https://user-images.githubusercontent.com/1496996/71522229-69454780-2889-11ea-8b00-a6c46deb9664.jpg)
